src/providers/node.rs

```diff
@@ -143,7 +190,30 @@ impl NodeProvider {
         Ok(false)
     }
 
-    pub fn get_start_cmd(app: &App) -> Result<Option<String>> {
+    pub fn get_start_cmd(app: &App, env: &Environment) -> Result<Option<String>> {
+        if NodeProvider::is_nx_monorepo(app) {
+            let output_path = NodeProvider::get_nx_output_path(app, env)?;
+            let project_json = NodeProvider::get_nx_project_json_for_app(app, env)?;
+
+            let executor = project_json.targets.build.executor;
+            if executor == "@nrwl/next:build" {
```

Comment:
A few comments for the NX start command

- The port shouldn't be set when running a Next app. It should just use the default
- Instead of cd'ing into the directory, I think it is cleaner to start the app with Nx. E.g. `nx run frontend:start`
- The default start command for all NX apps should be running the start command in the app (if it exists). For example for something like Yarn/NX monorepos, each sub-app will have it's own start command that we should use (if it exists). Then we can fallback to running `dist/main.js`

Reply:
Same as the other start commands.

Regarding the port.  The default port is something like 3200, I would expect a website to be hosted on port 80 and without this change, it would not be. removing this the user would have to overnight the start command manually to use port 80. 

Would the default port be acceptable?

Reply:
For Next apps the port is overridable with the `PORT` environment variable and that is common in many other frameworks. Also, when starting the container, you can map port 3200 inside of the container to any other port on the outside, so the default port in the container does not matter.

Reply:
Sounds good to me. I'll get on this change when I can.

Reply:
Should be done now. Just the run commands to change as of present.